docker-publish.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+if [ "A$1" != "Alocal" ]; then
+# required to push
+    docker login
```

Comment:
Would be nice to check if the login was successful before continuing.

Reply:
ok, I'll add it